packages/evolution-frontend/package.json

```diff
@@ -72,7 +72,8 @@
     "react-table": "^7.8.0",
     "redux": "^5.0.1",
     "redux-thunk": "^3.1.0",
-    "remark-gfm": "^4.0.0"
+    "remark-gfm": "^4.0.0",
+    "uuid": "^11.1.0"
```

Comment:
Je crois que ceci peut aller dans les devDependencies, c'est juste pour les tests unitaires iiuc

Reply:
moved uuid to devDep.